### PR TITLE
Fix for issue #692

### PR DESCRIPTION
**Pull request by mbudiu-vmw**

### CODE REVIEW

**Comment by mbudiu-vmw** [comment]
No one wants to review this one-line fix?

File: backends/bmv2/common/expression.cpp [hunk 1]
```diff
@@ -332,6 +332,8 @@ void ExpressionConverter::postorder(const IR::Member* expression)  {
     if (expression->expr->is<IR::Member>()) {
         auto mem = expression->expr->to<IR::Member>();
         auto memtype = typeMap->getType(mem->expr, true);
+        if (memtype->is<IR::Type_Stack>() && mem->member == IR::Type_Stack::next)
+            ::error("%1%: Reading next field, which is always uninitialized", mem);
```
Comment (jnfoster) [inline backends/bmv2/common/expression.cpp]: Is there a test case for this behavior?

Comment (mbudiu-vmw) [reply backends/bmv2/common/expression.cpp]: Yes, it is part of the PR.

Comment (mbudiu-vmw) [reply backends/bmv2/common/expression.cpp]: It's the file called issue692-bmv2.p4.
Note that this is an error only in this back-end.
It's a warning in the front-end.

Comment (jnfoster) [reply backends/bmv2/common/expression.cpp]: Got it.

**Pull request merged**